packages/cli/src/wrapper.ts

```diff
@@ -10,8 +10,14 @@ import {
 import { lintConfigCallback } from './commands/lint';
 import type { CommandOptions } from './types';
 
+type CommandHandler<T extends CommandOptions> = (
+  argv: T,
+  config: Config,
+  version: string
+) => Promise<void> | void;
+
 export function commandWrapper<T extends CommandOptions>(
-  commandHandler: (argv: T, config: Config, version: string) => Promise<void>
+  commandHandler: CommandHandler<T> = async () => {}
```

Comment:
Is there a need to specify the default function?

Reply:
Its workaround is to allow an empty function param `()=>{}`.
I prefer variant with optional parameter and check if we need to call `commandHandler`

Reply:
Or maybe we can change type to 
```
type CommandHandler<T extends CommandOptions> =
  | ((argv: T, config: Config, version: string) => Promise<void>)
  | ((argv: T) => void);
```